Doc/alert.html

```diff
@@ -0,0 +1,145 @@
+                </ul>
+            </div>
+            <h3>Components</h3>
+            <div class="list-wrap">
+                <ul>
+                    <a href="./alert.html"><li class="list-pills"><h4>Alert</h4></li></a>
+                    <a href="./avatar.html"><li class="list-pills">Avatar</li></a>
+                    <a href="./badge.html"><li class="list-pills">Badge</li></a>
+                    <a href="./button.html"><li class="list-pills">Button</li></a>
+                    <a href="./card.html"><li class="list-pills">Card</li></a>
+                    <a href="./image.html"><li class="list-pills">Image</li></a>
+                    <a href="./input.html"><li class="list-pills">Input</li></a>
+                    <a href="./text-utilites.html"><li class="list-pills">Text Utilities</li></a>
+                    <a href="./list.html"><li class="list-pills">list</li></a>
+                    <a href="./navigation.html"><li class="list-pills">Navigation</li></a>
+                    <a href="./modal.html"><li class="list-pills">Modal</li></a>
+                    <a href="./rating.html"><li class="list-pills">Rating</li></a>
+                    <a href="./snackbar.html"><li class="list-pills">Snackbar</li></a>
+                    <a href="./grid.html"><li class="list-pills">Grid</li></a>
+                </ul>
+            </div>
+        </div>
+        <!-- main content area -->
+
+        <div class="main-display ">
+                <h1 class="margin-top-bottom">Alert</h1>
+                <p class="simebold-font">This component is used to give an impotant message to the user on urgent basis. The message can be warning , info, update or status anythingwhich need user attention!</p> 
+
+                <h2 class="margin-top-bottom">Default Alert</h2>
+
+                <div class="component-display-div maple-flex-xy-center">
+                    <div class="alert">
+                        <h5 class="alert-message">This is the alert message you have notification</h5>
+                    </div>
+                    <div class="alert alert-dark">
+                        <h5 class="alert-message">This is the alert message you have notification</h5>
+                    </div>
+                </div>
+
+                <h2 class="margin-top-bottom">Attention (status) Alert</h2>
+
+                <div class="component-display-div maple-flex-xy-center">
+                    <div class="alert alert-success ">
+                        <i class="fas fa-check-square"></i>
+                        <h5 class="alert-message">This is the alert message you have notification</h5>
+                    </div>
+                    <div class="alert alert-fail">
+                        <i class="fas fa-bomb"></i>
+                        <h5 class="alert-message">This is the alert message you have notification</h5>
+                    </div>
+                    <div class="alert  alert-warn">
```

Comment:
aha, found a 2 space difference between classes-
refactor the code properly-

Reply:
removed extra space 😅